doc/getting-started-guide.md

```diff
@@ -0,0 +1,245 @@
+# Getting Started Guide
+
+## 1. Deploy the EdgeX
+Run core-data, core-metadata, core-command service according to the EdgeX doc
+https://docs.edgexfoundry.org/2.0/getting-started/Ch-GettingStartedGoDevelopers/.
```

Comment:
Do we want to point to Jakarta? https://docs.edgexfoundry.org/2.1/getting-started/Ch-GettingStartedGoDevelopers/

Reply:
@lenny-intel Is Kamakura more properly?

Reply:
Yes, since that will be the target release for these camera services. Good catch. 
Once accepted into the main org the service will be update to use latest Kamakura SDK and hopefully release with the Core Service... ;-)

Reply:
Thanks @lenny-intel and @weichou1229!

Reply:
Updated the link.